packages/tasks/src/solanaDeposit.ts

```diff
@@ -23,7 +30,7 @@ export const solanaDeposit = async (
   }
   const { amount, api, idPath } = args;
   const params = [JSON.parse(args.types), args.values];
-  await client.solanaDeposit({ amount, api, idPath, params, recipient });
+  await client.solanaDeposit({ amount, api, params, recipient });
```

Comment:
I don't think passing an API with every request is optimal. I think we should just read the API from the networks package (v10.0.0-rc1 for now): https://github.com/zeta-chain/networks/blob/c4f15dd424890296f67a1913038a11364130382f/data/networks.json#L740

Reply:
Done. And maybe we should add `solana` to the ApiType: https://github.com/zeta-chain/networks/blob/c4f15dd424890296f67a1913038a11364130382f/src/types.ts#L8